core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/validation/upload/UploadersValidator.java

```diff
@@ -41,7 +41,7 @@ public final class UploadersValidator {
     private UploadersValidator() {
         // noop
     }
-
+    // TODO: viqueen - we need a validateBitbucketCloudUploader
```

Comment:
Only if BitbucketCloud supports uploading arbitrary files to a given location (storage)